### DISCUSSION TITLE
Quick Preview Pane - Ability to hide status bar

### DISCUSSION THREAD

**Discussion by advice2020** [doublecmd/doublecmd#1597] [Suggestions]
Quick Preview pane shows status bar at the bottom.

Suggesting to offer a setting that would allow user the ability to choose to hide this status bar from being viewed when using QPP.

Some opinions/reasons for this requests...
-Status bar can show a lot of text that is not necessarily needed to be seen, especially in QPP mode.
-This text can blend into certain WLX plugin displays/panes.
-Allows QPP to have a simple/clean pane display showing only the current item being previewed.
-Typically QPP will be about 50% of screen which means the content in the status bar gets cut off anyways / can look strange to see truncated text the abruptly ends.
-QPP mode shows preview based on which folder/file is current highlighted in file pane based on its MIME type.  Users will most likely set which plugin to use to preview a specific MIME type, user will not really require need to change this in status bars context menu in this use case.

As mentioned, this would be in the form of a setting where user can choose to show the status bar or not.  Default behavior would remain.
Setting could be made to...
-Hide status bar completely
or if easier to implement
-Show as blank status bar with no text inside of it.

Hopefully something like this will be considered to be updated.
Thank you to anyone who reads this

**Comment by j2969719** [comment]
in the latest versions of doublecmd, right-clicking on this status bar allows you to change the viewing mode

**Comment by advice2020** [comment]
@j2969719 
Thank you for your response.
I apologize if I am not understanding correctly.  I have a status bar context menu but there isn't a way to turn off the status bar from it.  I am requesting a setting that would allow user to be able to hide the status bar completely from the QPP.

**Reply by j2969719** [reply]
just simply meant that this menu will not be available without a statusbar

**Reply by advice2020** [reply]
-Would be a setting where user can choose to show the status bar or not.
-Not sure how other users view this, but to me, QPP WLX plugins are set to open specific MIME types without the need to change the viewer.

Just noticed the date, not sure which is stranger, that I posted this 2 years ago, or that it is as if I am posting the next day response.
Guess it didn't matter either way.

**Reply by advice2020** [reply]
Still interested if there are any developers also interested in this idea.
Thanks again
